cloud/kubernetes/helm/README.md

```diff
@@ -6,4 +6,4 @@ This chart installs Business Events application deployment on a [Kubernetes](htt
 
 ## Documentation
 
-`How to deploy TIBCO BusinessEvents® application` in kuberenetes using `helm` is available at [wiki pages](https://github.com/TIBCOSoftware/be-tools/wiki).
\ No newline at end of file
+`How to deploy TIBCO BusinessEvents® application` in kuberenetes using `helm` is available at [wiki pages](https://github.com/TIBCOSoftware/be-tools/wiki/Helm%20Chart).
```

Comment:
@LakshmiMekala making _[wiki pages]_ -> _[Helm Chart]_ would be good

Reply:
text updated